tuf/repository_lib.py

```diff
@@ -146,9 +146,12 @@ def _generate_and_write_metadata(rolename, metadata_filename,
       _log_warning_if_expires_soon(TARGETS_FILENAME, roleinfo['expires'],
           TARGETS_EXPIRES_WARN_SECONDS)
 
+    # Don't hash-prefix consistent target files if they are handled out of band
+    consistent_targets = consistent_snapshot and not use_existing_fileinfo
```

Comment:
Could we please say `write_consistent_targets`? (Probably applies to `consistent_snapshot`, too, which should be renamed to `write_consistent_metadata`, I guess.)

Reply:
I was torn there. Fitting it on one line + consistency with `consistent_targets` vs. more mnemonic name + consistency with the arg name  in `generate_targets_metadata`. :)

I'm fine either way.

Reply:
I have no strong feelings, and am happy to go either way, too.